app/code/Bulbulatory/Recommendations/view/frontend/layout/customer_account.xml

```diff
@@ -0,0 +1,12 @@
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="customer_account_navigation">
+            <block class="Magento\Framework\View\Element\Html\Link\Current" name="customer-account-navigation-custom">

```

Comment:
This new tab will be visible even if the module is disabled in configuration. You can add attribute `ifconfig="..."` with a path to the "enabled" configuration field so it will only show it when the module is enabled.

Reply:
@TM18

> This new tab will be visible even if the module is disabled in configuration. You can add attribute ifconfig="..." with a path to the "enabled" configuration field so it will only show it when the module is enabled.

Fixed!